README.md

```diff
@@ -462,73 +462,71 @@ You're already using Nextcloud and want to switch to docker? Great! Here are som
     docker-compose exec db sh -c "psql -U USER --set ON_ERROR_STOP=on nextcloud < /dmp"
     docker-compose exec db rm /dmp
     ```
 3. Edit your config.php
-  1. Set database connection
-     - In case of MySQL database
-      ```php
-      'dbhost' => 'db:3306',
-      ```
-     - In case of PostgreSQL database
-      ```php
-      'dbhost' => 'db:5432',
-      ```
-  2. Make sure you have no configuration for the `apps_paths`. Delete lines like these
-  ```diff
-  - 'apps_paths' => array (
-  -     0 => array (
-  -         'path' => OC::$SERVERROOT.'/apps',
-  -         'url' => '/apps',
-  -         'writable' => true,
-  -     ),
-  - ),
-  ```
-  3. Make sure to have the `apps` directory non writable and the `custom_apps` directory writable
-  ```php
-  'apps_paths' => array (
-    0 => array (
-      'path' => '/var/www/html/apps',
-      'url' => '/apps',
-      'writable' => false,
-    ),
-    1 => array (
-      'path' => '/var/www/html/custom_apps',
-      'url' => '/custom_apps',
-      'writable' => true,
-    ),
-  ),
-  ```
-  4. Make sure your data directory is set to /var/www/html/data
-  ```php
-  'datadirectory' => '/var/www/html/data',
-  ```
-
-
+    1. Set database connection
+        - In case of MySQL database
+        ```php
+        'dbhost' => 'db:3306',
+        ```
+        - In case of PostgreSQL database
+        ```php
+        'dbhost' => 'db:5432',
+        ```
+    2. Make sure you have no configuration for the `apps_paths`. Delete lines like these
+        ```diff
+          - 'apps_paths' => array (
+          -     0 => array (
+          -         'path' => OC::$SERVERROOT.'/apps',
+          -         'url' => '/apps',
+          -         'writable' => true,
+          -     ),
+          - ),
+          ```
+    3. Make sure to have the `apps` directory non writable and the `custom_apps` directory writable
+        ```php
+        'apps_paths' => array (
+          0 => array (
+            'path' => '/var/www/html/apps',
+            'url' => '/apps',
+            'writable' => false,
+          ),
+          1 => array (
+            'path' => '/var/www/html/custom_apps',
+            'url' => '/custom_apps',
+            'writable' => true,
+          ),
+        ),
+        ```
+    4. Make sure your data directory is set to /var/www/html/data
+        ```php
+        'datadirectory' => '/var/www/html/data',
+        ```
 4. Copy your data (nextcloud_app_1 is the name of your Nextcloud container):
-```console
-docker cp ./data/ nextcloud_app_1:/var/www/html/
-docker-compose exec app chown -R www-data:www-data /var/www/html/data
-docker cp ./theming/ nextcloud_app_1:/var/www/html/
-docker-compose exec app chown -R www-data:www-data /var/www/html/theming
-docker cp ./config/config.php nextcloud_app_1:/var/www/html/config
-docker-compose exec app chown -R www-data:www-data /var/www/html/config
-```
+    ```console
+    docker cp --preserve ./data/ nextcloud_app_1:/var/www/html/
```

Comment:
I was unable to find docs for this flag . Shouldn't it be documented here? https://docs.docker.com/engine/reference/commandline/cp/

Reply:
My bad. Because docker cp doesn't support copying with preserving modification date, I copied the data with the system's cp into the named volume, which supports the flag. When I prepared this PR from my notes of the migration process, I overlooked that the original doc suggested to use docker cp.  

0d4148f9a490aa3177cb00d1037c4c868e00f64b rewrites this as an alternative suggestion, making it clearer that this doesn't involve docker's cp.

Reply:
Alternatively, it could be something like this, making source and destination clearer:
```bash
cp --preserve --recursive /pathToOldNextcloud/data/ /pathToNextcloudDockerVolume/data
```

Reply:
@tilosp do you prefer the version from 0d4148f9a490aa3177cb00d1037c4c868e00f64b or should I give it another spin with the suggestion from https://github.com/nextcloud/docker/pull/1383#discussion_r563952856?